File: PythonParser/Target.py
class Target(object):
    def __init__(self, parse_string):
        self.strings = parse_string

    def is_equal(self, comparison):
        if (self.strings == comparison.strings):
            return True
        else:
            return False

    def is_bigger(self, comparison):
        if (int(self.strings) >= int(comparison.strings)):
            return True
        else:
            return False

    def extract(self, last_index, index):
        return Target(self.strings[last_index:index])

File: RubyParser/target.rb
class Target
    attr_accessor :strings

    def initialize(parse_string)
      @strings = parse_string
    end

    def is_equal(comparison)
      if self.strings == comparison.strings
        return true
      else
        return false
      end
    end

    def is_bigger(comparison)
      if self.strings >= comparison.strings
        return true
      else
        return false
      end
    end

    def extract(last_index, chara_num = 1)
      return Target.new(self.strings[last_index, chara_num])
    end
end